shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Delegs.hs

```diff
@@ -70,7 +70,7 @@ data DELEGS era
 data DelegsEnv era = DelegsEnv
   { delegsSlotNo :: SlotNo,
     delegsIx :: Ix,
-    delegspp :: PParams,
+    delegspp :: (PParams era),
```

Comment:
These parenthesis are redundant (same for the line below), I suspect a result of using find & replace.